src/test/scala/scala/tools/refactoring/tests/implementations/MarkOccurrencesTest.scala

```diff
@@ -439,4 +438,74 @@ class MarkOccurrencesTest extends TestHelper {
       def doit = for (#(x) <- Seq(#(1), #(2))) yield x
     }
     """)
+
+  @Test
+  def trivialObject() = markOccurrences("""
+    object DasDing/*<-cursor-3*/
+    """, """
+    object #######/*<-cursor-3*/
+    """)
+
+  @Test
+  def nextToTrivialObject() = markOccurrences("""
+    object /*<-*/KnappDaneben
+    """, """
+    object /*<-*/KnappDaneben
+    """)
+
+  @Test
+  def onObjectKeyword() = markOccurrences("""
+    object/*<-cursor*/ WiederVorbei
+    """, """
+    object/*<-cursor*/ WiederVorbei
+    """)
+
+  @Test
+  def onEndOfBacktickAbomination() = markOccurrences("""
+    object `/*<!!-<.>-!!>*/`/*<-cursor*/
+    """, """
+    object #################/*<-cursor*/
+    """)
+
+  @Test
+  def onStartOfBacktickAbomination() = markOccurrences("""
+    object /*cursor->*/`/*<!!-<.>-!!>*/`
+    """, """
+    object /*cursor->*/#################
+    """)
+
+  @Test
+  def inTheCenterOfBacktickAbomination() = markOccurrences("""
+    object /*8-cursor->*/`/*<!!-<.>-!!>*/`
+    """, """
+    object /*8-cursor->*/#################
+    """)
+
+   @Test
+   def onPlusOperator() = markOccurrences("""
+     object Abacus {
+       val a = 4 +/*<-cursor*/ 3
+       val b = a * 5 + 10
+     }
+   """, """
+     object Abacus {
+       val a = 4 #/*<-cursor*/ 3
+       val b = a * 5 # 10
+     }
+   """)
+
+   @Test
+   def onConsOperator() = markOccurrences("""
+     object Constanzia {
+       val a = 1 :: 2 :: 3 :: Nil
+       val b = '1' :: '2' :: Nil
+       val c = 1.1 :: 2.2 ::/*<-cursor*/ Nil
+     }
+   """, """
+     object Constanzia {
+       val a = 1 ## 2 ## 3 ## Nil
```

Comment:
No, it doesn't, although the unit tests prove that the problem lies not within the refactoring library (at least not any more). I'll file an IDE ticket for this as soon as this PR is merged.